core/controllers/profile_test.py

```diff
@@ -312,3 +312,60 @@ def test_contribution_datetime(self):
         self.assertEqual(
             response_dict['first_contribution_datetime'],
             utils.get_time_in_millisecs(current_datetime))
+
+
+class UserContributionsTests(test_utils.GenericTestBase):
+
+    USERNAME_A = 'a'
+    EMAIL_A = 'a@example.com'
+    USERNAME_B = 'b'
+    EMAIL_B = 'b@example.com'
+    EXP_ID_1 = 'exp_id_1'
+
+    def test_zero_count(self):
+        #Check to see a user with no contributions
+        #shows up as 0 created and edited on the profile page.
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.login(self.EMAIL_A)
```

Comment:
Either log out at the end, or don't log in at all (which is fine since the profile pages are world-viewable).

Ditto for the other tests.